backend/gn_plugin_depobio/demarches_simplifiees/folder.py

```diff
@@ -0,0 +1,107 @@
+import dataclasses
+import datetime
+from enum import Enum
+from typing import Optional
+from geonature.utils.config import config
+
+configuration_depobio = config["PLUGIN_DEPOBIO"]
+
+
+class State(str, Enum):
+    EN_CONSTRUCTION = "en_construction"
+    ACCEPTE = "accepte"
+    EN_INSTRUCTION = "en_instruction"
+    REFUSE = "refuse"
+    SANS_SUITE = "sans_suite"
+
+
+@dataclasses.dataclass()
+class Field:
+    id: str
+    champDescriptorId: str
+    _typename: str
+    label: str
+    stringValue: str
+    prefilled: bool
+    columns: list
+    values: str = None
```

Comment:
```suggestion
    values: List[str] = None
```

D'après les exemples qu'on a dans "api_responses.py".